docs/VnVPlan/VnVPlan.tex

```diff
@@ -69,141 +70,158 @@ \section{Symbols, Abbreviations, and Acronyms}
+\end{itemize}
 
 \section{Plan}
 
-\wss{Introduce this section.   You can provide a roadmap of the sections to
-  come.}
+The following will outline a series of plans and processes that will be used for the verification and validation of the software solution and documentation the Grocery Spending Tracker project. The sections are as follows:
+\begin{itemize}
+    \item Verification and Validation Team: the team members their involvement in the verification and validation efforts 
+    \item SRS Verification Plan: the approaches to be used for SRS verification
+    \item Design Verification Plan: plans for design verification
+    \item Verification and Validation Plan Verification Plan: plans for VnV verification
+    \item Implementation Verification Plan: dynamic and static tests for verification of the implementation
+    \item Automated Testing and Verification Tools: tools and metrics to be used for testing and verification of the implementation
+    \item Software Validation Plan: plans for verification of the software requirements
+\end{itemize}
+The stakeholders referred to in this document are individuals who are members of low-income households in Hamilton and students from McMaster University as described in Section 2 of the \href{https://github.com/r-yeh/grocery-spending-tracker/blob/master/docs/SRS/SRS.pdf}{SRS}. As the primary client, their involvement in the VnV ensures that the system aligns with the needs and goals of the intended users.
 
 \subsection{Verification and Validation Team}
-
-\wss{Your teammates.  Maybe your supervisor.
-  You should do more than list names.  You should say what each person's role is
-  for the project's verification.  A table is a good way to summarize this information.}
+\begin{table}[H]
+    \centering
+    \begin{tabularx}{\textwidth}{|>{\hsize=.36\hsize}X|>{\hsize=1.64\hsize}X|}
+        \hline
+        \textbf{Name} & \textbf{Role}\\
+        \hline
+        Allan Fang & Developer: \\
+        \hline
+        Jason Nam & Developer: \\
+        \hline
+        Ryan Yeh & Developer: \\
+        \hline
+        Sawyer Tang & Developer: \\
+        \hline
+    \end{tabularx}
+    \caption{VnV Team}
+    \label{tab:vnvteam}
+\end{table}
 
 \subsection{SRS Verification Plan}
-
-\wss{List any approaches you intend to use for SRS verification.  This may include
-  ad hoc feedback from reviewers, like your classmates, or you may plan for 
-  something more rigorous/systematic.}
-
-\wss{Maybe create an SRS checklist?}
+The design documentation will be verified with an informal walk-through by stakeholders and peers of the SFWRENG 4G06 course who will provide ad-hoc review as well as an inspection by the development team. The inspection will address the following issues and be tracked by the checklist:
+\begin{itemize}
+    \item Correctness: the SRS accurately represents the software expectations
```

Comment:
Can you link the SRS here too? I think for each subsection/section, if we mention a document, we should at least hyperlink it once. Don't need to do it for the other instance of SRS in this subsection/

Reply:
yur